src/JavascriptErrorContext.php

```
@@ -0,0 +1,129 @@
+use Behat\Behat\Hook\Scope\AfterScenarioScope;
+use Behat\MinkExtension\Context\MinkContext;
+
+/**
+ * Tracks javascript errors on the page.
+ */
+class JavascriptErrorContext extends MinkContext implements Context {
+
+  /**
+   * Errors registry keyed by path.
+   */
+  private array $jsErrorRegistry = [];
```

PHPStan (2.1.15)

Property DrevOps\BehatSteps\JavascriptErrorContext::$jsErrorRegistry type has no value type specified in iterable type array.
See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type

(missingType.iterableValue)